<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="PavanKalyan" failures="1" tests="3" name="Default test(failed)" time="24.943" errors="0" timestamp="2024-05-29T10:42:45 IST">
  <testcase classname="com.vTiger.crm.InvoiceTest.InvoiceTest" name="createInvoiceTest" time="0.052">
    <skipped/>
  </testcase> <!-- createInvoiceTest -->
  <testcase classname="com.vTiger.crm.InvoiceTest.InvoiceTest" name="createInvoiceTest" time="0.005">
    <skipped/>
  </testcase> <!-- createInvoiceTest -->
  <testcase classname="com.vTiger.crm.InvoiceTest.InvoiceTest" name="createInvoiceTest" time="0.019">
    <failure type="java.lang.AssertionError" message="expected [Login] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [Login] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]
at com.vTiger.crm.InvoiceTest.InvoiceTest.createInvoiceTest(InvoiceTest.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- createInvoiceTest -->
  <testcase classname="com.vTiger.crm.InvoiceTest.InvoiceTest" name="createInvoiceWithOrgTest" time="0.0">
    <ignored/>
  </testcase> <!-- createInvoiceWithOrgTest -->
</testsuite> <!-- Default test(failed) -->
